src/CompanyService.Models.Broker/Models/UserData.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace LT.DigitalOffice.UserService.Models.Broker.Models
+{
+    public class UserData
```

Comment:
Why did you use class, why not interface?

Reply:
With interface not work

Reply:
Why is it not work?